content/3_Silver/Conclusion.problems.json

```
@@ -241,6 +241,19 @@
         "site": "CF"
       }
     },
+    {
+      "uniqueId": "cf-1943C",
+      "name": "Tree Compass",
+      "url": "https://codeforces.com/contest/1943/problem/C",
+      "source": "CF",
+      "difficulty": "Normal",
```

```suggestion
      "difficulty": "Very Hard",
```
difficulty adjustment

---

太阳从西边出来了